packages/next/client/image.tsx

```diff
@@ -228,14 +228,16 @@ function removePlaceholder(
   placeholder: PlaceholderValue
 ) {
   if (placeholder === 'blur' && element) {
-    if (element.complete) {
+    if (element.complete && !element.src.startsWith('data:')) {
```

Comment:
Can you explain a bit more about your reasoning behind checking the src?

Reply:
When `isVisible === false`, the value of `src` always starts with `data:`.